zone_validate.py

```diff
@@ -65,6 +65,10 @@ def is_valid_name(name, strict=True):
 
 def is_valid_target(name, strict=False):
     """Check that the target of a RR is valid"""
+    
+    if name[0] in string.digits:
+        name = "d" + name[1:]
+        
```

Comment:
This causes an exception if the target is of length zero, which is the case for some tests such as:

test_is_valid_mx_one_empty

Reply:
Second issue is that some targets are intentionally a single digit, e.g. SRV records. Prepending 'd' breaks the tests:

test_is_valid_zone_record_srv_bad